python/test/t_IndependentCopulaFactory_std.py

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env python
+
+from openturns import *
```

Comment:
Could you also write t_IndependentCopulaFactory as a C++ test?

Reply:
Also, "from openturns import *" should be replaced with "import openturns as ot" which is cleaner.